### DISCUSSION TITLE
Pixelfed on NixOS

### DISCUSSION THREAD

**Discussion by nerdbude** [pixelfed/pixelfed#5253] [Q&A]
I started configuring Pixelfed on my NixOS server. 
I went through the steps described in other tutorials (adapted to NixOS). 
I have Pixelfed at `/var/www/pixelfed` with permissions `pixelfed:pixelfed` on all directories and files. 
Redis, mysql and nginx are up and running. Unfortunately, nginx throws the following error: 
`directory index of "/var/www/pixelfed/public" is forbidden`. 
When I access the domain via browser I also get the error 403 forbidden (when I call domain.tld/offline.html it is displayed correctly but reloads every second). 
Does anyone know how I can fix the permission error?

**Comment by Radvendii** [comment]
You might be running into the issue described here https://github.com/NixOS/nixpkgs/issues/235147

In my case, it was just the images not displaying; the rest of the page loaded.
